web-console/src/druid-models/native-json-query/native-json-query-completions.ts

```diff
@@ -0,0 +1,1098 @@
+      { value: 'stringAny', documentation: 'Any value aggregator for strings' },
+      {
+        value: 'thetaSketch',
+        documentation: 'Theta sketch for approximate distinct count',
+      },
+      {
+        value: 'HLLSketchBuild',
+        documentation: 'HLL sketch for approximate distinct count',
+      },
+      {
+        value: 'quantilesDoublesSketch',
+        documentation: 'Quantiles sketch for doubles',
+      },
+      {
+        value: 'hyperUnique',
+        documentation: 'HyperLogLog for approximate distinct count',
+      },
+      { value: 'cardinality', documentation: 'Cardinality aggregator' },
+      { value: 'histogram', documentation: 'Approximate histogram aggregator' },
+      {
+        value: 'fixedBucketsHistogram',
+        documentation: 'Fixed buckets histogram aggregator',
+      },
+      { value: 'filtered', documentation: 'Filtered aggregator' },
+      { value: 'grouping', documentation: 'Grouping aggregator for subtotals' },
+    ],
+  },
+
+  // Aggregation properties for field-based aggregators
+  {
+    path: '$.aggregations.[]',
+    isObject: true,
+    condition: obj =>
+      obj.type &&
+      [
+        'longSum',
+        'doubleSum',
+        'floatSum',
+        'longMin',
+        'doubleMin',
+        'floatMin',
+        'longMax',
+        'doubleMax',
+        'floatMax',
+        'doubleMean',
+      ].includes(obj.type),
+    completions: [
+      { value: 'fieldName', documentation: 'The field to aggregate' },
+      { value: 'expression', documentation: 'Expression to evaluate instead of fieldName' },
+    ],
+  },
+
+  // Aggregation properties for first/last aggregators
+  {
+    path: '$.aggregations.[]',
+    isObject: true,
+    condition: obj =>
+      obj.type &&
+      [
+        'doubleFirst',
+        'doubleLast',
+        'floatFirst',
+        'floatLast',
+        'longFirst',
+        'longLast',
+        'stringFirst',
+        'stringLast',
+        'doubleAny',
+        'floatAny',
+        'longAny',
+        'stringAny',
+      ].includes(obj.type),
+    completions: [
+      { value: 'fieldName', documentation: 'The field to aggregate' },
+      {
+        value: 'timeColumn',
+        documentation: 'Time column to use for ordering (defaults to __time)',
+      },
+    ],
+  },
+
+  // Filtered aggregator properties
+  {
+    path: '$.aggregations.[]',
+    isObject: true,
+    condition: obj => obj.type === 'filtered',
+    completions: [
+      { value: 'filter', documentation: 'Filter to apply before aggregation' },
+      { value: 'aggregator', documentation: 'The aggregator to apply to filtered data' },
+    ],
+  },
+
+  // Filter types
+  {
+    path: '$.filter',
+    isObject: true,
+    completions: [{ value: 'type', documentation: 'Type of filter' }],
+  },
+
+  {
+    path: '$.filter.type',
+    completions: [
+      { value: 'selector', documentation: 'Matches a specific dimension value' },
```

Comment:
this is missing the [range filter](https://druid.apache.org/docs/latest/querying/filters/#range-filter) and typed in filter (which i just realized isn't documented yet, from #16039)

Reply:
I am sticking only to the documented stuff for now